pabi_account_report/reports/xlsx_report_expense_ledger.xml

```diff
@@ -14,7 +14,7 @@
                 <xpath expr="//notebook[@name='additional_criteria']" position="inside">
                     <page string="Account Filters" name="account_filters">
                         <separator string="Print only" colspan="4"/>
-                        <field name="account_ids" domain="[('company_id', '=', company_id), ('user_type', '=', user_type)]"/>
+                        <field name="account_ids" domain="[('company_id', '=', company_id), ('user_type', 'in', (9,33))]"/>
```

Comment:
มีวิธีอื่นที่ไม่ต้อง hard code ไม๊ครับ

Reply:
ใช้เป็นตัวนี้แทนได้ไหมครับ
"<field name="account_ids" domain="[('company_id', '=', company_id), ('user_type.code', 'in', (Expense, Allocation)))]"/>"

Reply:
ดีกว่าครับ อ่านแล้วเข้าใจ

Reply:
Update xlsx_report_expense_ledger.xml
change 9,33 to ('Expense', 'Allocation')